public/src/main/java/com/revenuecat/purchases/EntitlementInfo.kt

```diff
@@ -40,7 +46,12 @@ data class EntitlementInfo constructor(
     val unsubscribeDetectedAt: Date?,
     val billingIssueDetectedAt: Date?,
     val ownershipType: OwnershipType,
-) : Parcelable {
+    val jsonObject: JSONObject
```

Comment:
We should also add the `@Deprecated` here

Reply:
Done, thanks!